### PR TITLE
Fix: edit dates in stats section

### PR DESCRIPTION
**Pull request by teoh4770**
Issue: #197 

- [X] I update the image that contains the wrong date in desktop view
- [X] I update the correct date(text) in mobile view

Note: let me know if the background should be larger like the previous one

### CODE REVIEW

File: src/assets/hero-about/hero-about-desktop.svg
Comment (juancwu) [inline src/assets/hero-about/hero-about-desktop.svg:1]: The height is diff, just want to make sure that this is not causing any issues.

Comment (teoh4770) [reply src/assets/hero-about/hero-about-desktop.svg:1]: Is not causing any issue, but let me double check

Comment (teoh4770) [reply src/assets/hero-about/hero-about-desktop.svg:1]: let me try to import the larger one to see the difference

Comment (aidantrabs) [reply src/assets/hero-about/hero-about-desktop.svg:1]: lmk if all is good after testing and ill merge

Comment (teoh4770) [reply src/assets/hero-about/hero-about-desktop.svg:1]: the larger one is better...let me change it

Comment (teoh4770) [reply src/assets/hero-about/hero-about-desktop.svg:1]: done

**Review by GabeDiniz — APPROVED** [review]

**Review by juancwu — APPROVED** [review]

**Pull request merged**